extensions/github-review-requests/package.json

```diff
@@ -32,34 +37,45 @@
       "required": true,
       "title": "Personal Access Token",
       "description": "Create a token with repository access"
+    },
+    {
+      "name": "owners",
+      "type": "textfield",
+      "required": false,
+      "title": "Organizations/Owners",
+      "description": "Please specify the organizations or owners you want to focus on, separating them by commas.",
+      "placeholder": "(Optional) Include all orgs or owners."
```

Comment:
Maybe, somehow reword it, signifying that it's okay to leave it empty, and in this case, one would get all owners in the response.